roles/servicemesh/tasks/main.yml

```diff
@@ -0,0 +1,343 @@
+---
+- name: Get current state
```

Comment:
I wonder if we can test to see if `Kind: ServiceMesh , ServiceMeshMemberRoll ` is even something we can create before we jump through the tasks... To catch if the user has not deployed ServiceMesh?

Reply:
Failing fast is definitely a good thing. Do you know if the k8s_info allows you to query if certain CRD is installed?

Reply:
Something like this should work:
```yaml
- name: Get CRDs
  k8s_info:
    api_version: apiextensions.k8s.io/v1
    kind: CustomResourceDefinition
    name: apiservers.config.openshift.io
  register: apiservers
```